docs/single-to-multi-master.md

```diff
@@ -30,7 +30,7 @@ Please follow all the backup steps before attempting it. Please read the
 [etcd admin guide](https://github.com/coreos/etcd/blob/v2.2.1/Documentation/admin_guide.md)
 before attempting it.
 
-We can migrate from a single-master cluster to a multi-master cluster, but this is a complicated operation. It is easier to create a multi-master cluster using Kops (described [here](https://github.com/kubernetes/kops/blob/master/docs/high_availability.md)). If possible, try to plan this at time of cluster creation.
+We can migrate from a single-master cluster to a multi-master cluster, but this is a complicated operation. It is easier to create a multi-master cluster using Kops (described [here](https://github.com/kubernetes/kops/blob/master/docs/single-to-multi-master.md)). If possible, try to plan this at time of cluster creation.
```

Comment:
I think `high_availability.md` was moved into an `operations` subdirectory here:

https://github.com/kubernetes/kops/pull/7593/files#diff-e089e6f8aa55d07a58d6e6001cea4b32

so we could probably update this link to be a relative path: `./operations/high_availability.md`

Reply:
Fixing the `high_availability.md` link to be a correct relative path is #7907

Reply:
Ok, I will solve this conflict.
But I suggest:
`operations/high_availability.md` -> `https://github.com/kubernetes/kops/blob/master/docs/operations/high_availability.md`

Reply:
It would be more effective to make that suggestion in the other PR. I currently disagree with the suggestion, but providing the reasoning behind the suggestion might help lead to a common understanding.

Reply:
@johnbelamaric 
https://github.com/kubernetes/kops/blob/master/docs/operations/high_availability.md is right
this is hyperlink

Reply:
@johnbelamaric 
both are ok. But I think hyperlink is better.

Reply:
I think the universal format is more readable.

Reply:
They're both hyperlinks. The absolute one has the disadvantage that if one is viewing from a tag or branch, the link will go to the tip of master branch, not to the version relevant to where one started. The destination page might even no longer exist on the master branch.